deploy/addons/nvidia-device-plugin/nvidia-device-plugin.yaml.tmpl

```diff
@@ -29,6 +29,10 @@ spec:
         name: nvidia-device-plugin-ds
     spec:
       tolerations:
+      - operator: Exists
+        effect: NoExecute
+      - operator: Exists
+        effect: NoSchedule
```

Comment:
Curious where you got these YAML changes from

Reply:
From /deploy/addons/gpu/nvidia-gpu-device-plugin.yaml.tmpl - https://github.com/kubernetes/minikube/blob/a0f915eda8a31049eb26e6eaa3dd0133e507bce0/deploy/addons/gpu/nvidia-gpu-device-plugin.yaml.tmpl#L35

Reply:
That config file is very old, those values are likely no longer needed when using the updated image. I think we should leave the config as is (remove the added values).

Reply:
Ok. Done.